app/assets/stylesheets/_subnav.scss

```diff
@@ -26,3 +26,86 @@
     }
   }
 }
+
+.study-subnav {
+  position: relative;
+  background-color: $color-off-white;
+  @include gradient-horizontal(
+    $start-color: $color-pale-green,
+    $end-color: $color-off-white,
+    $start-percent: 50%,
+    $end-percent: 50%
+  );
+  color: #aaa;
+
+  @media(max-width: $screen-xs-max){
+    display: none;
+  }
+
+  // Semi-transparent bottom border that sits on top of the
+  // coloured segments (saves giving each segment its own
+  // border, which would be tricky with the triangle ends).
+  &:after {
+    content: "";
```

Comment:
Properties should be ordered background, bottom, content, display, height, left, position, right